server/src/main/java/com/hedera/block/server/producer/ProducerBlockItemObserver.java

```diff
@@ -16,11 +16,16 @@
 
 package com.hedera.block.server.producer;
 
-import static com.hedera.block.protos.BlockStreamService.*;
-import static com.hedera.block.protos.BlockStreamService.PublishStreamResponse.*;
+import static com.hedera.block.server.Translator.toPbjBlockItem;
+import static com.hedera.block.server.Translator.toProtocPublishStreamResponse;
+import static com.hedera.block.server.producer.Util.getFakeHash;
 
 import com.hedera.block.server.ServiceStatus;
 import com.hedera.block.server.mediator.Publisher;
+import com.hedera.hapi.block.*;
```

Comment:
We should avoid wildcard imports.
If the IDE is doing this, setting the threshold to 99 usually works to stop it.